django_mailbox/models.py

```diff
@@ -498,10 +498,11 @@ def reply(self, message):
         pre-set it.
 
         """
-        if self.mailbox.from_email:
-            message.from_email = self.mailbox.from_email
-        else:
-            message.from_email = settings.DEFAULT_FROM_EMAIL
+        if not getattr(message, 'from_email', None):
```

Comment:
Ahh -- you're right; hrm, that makes me wonder why we're using `getattr` at all, though; really, `message.from_email` should be good enough (`None` is Falsy), but I can make the change while merging.

Also, FYI, the third parameter of `getattr` defaults to `None`; you need not specify it in situations where you'd like `None` returned when a given attribute does not exist.